Android build error

On android build i got this error message

`Error: /Users/vomchik/Projects/LITS/toolook/toolook-app/platforms/android/gradlew: Command failed with exit code 1 Error output:
/Users/vomchik/Projects/LITS/toolook/toolook-app/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						^
  symbol: class JavaScriptException
/Users/vomchik/Projects/LITS/toolook/toolook-app/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));
						                             ^
  symbol: class JavaScriptException
`

I've also had the same problem:

PATHREMOVED/cordova/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
13:12:43.774 						JavaScriptException ex = new JavaScriptException(data.getString(0));
13:12:43.774 						^
13:12:43.774   symbol: class JavaScriptException
13:12:43.777 PATHREMOVED/cordova/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
13:12:43.777 						JavaScriptException ex = new JavaScriptException(data.getString(0));
13:12:43.777 						                             ^
13:12:43.777   symbol: class JavaScriptException

confirmed => ionicApp/platforms/android/src/com/sarriaroman/fabric/FabricPlugin.java:142: error: cannot find symbol
						JavaScriptException ex = new JavaScriptException(data.getString(0));

Why doe this not help in my case? I still get `FabricPlugin.java:142: error: cannot find symbol  JavaScriptException ex = new JavaScriptException(data.getString(0));` while using ` <plugin name="cordova-fabric-plugin" spec="https://github.com/sarriaroman/FabricPlugin#1.1.1">`

--- a/plugin.xml
+++ b/plugin.xml
@@ -43,6 +43,7 @@
         </config-file>
 
         <source-file src="src/android/FabricPlugin.java" target-dir="src/com/sarriaroman/fabric"/>
+        <source-file src="src/android/JavaScriptException.java" target-dir="src/com/sarriaroman/fabric"/>
 
         <framework src="src/android/build-extras.gradle" custom="true" type="gradleReference" />
     </platform>

